crates/common/download/Cargo.toml

```diff
@@ -24,10 +24,11 @@ tokio = { version = "1.23", features = ["fs"] }
 url = "2.2"
 
 [dev-dependencies]
-anyhow = "1.0"
+anyhow = { version = "1.0", features = ["backtrace"] }
+env_logger = "0.10.0"
 mockito = "0.31"
 regex = "1.5"
 tempfile = "3.2"
 test-case = "2.2"
-tokio = { version = "1.23", features = ["macros"] }
+tokio = { version = "1.23", features = ["macros", "rt-multi-thread"] }
```

Comment:
What is the reason for using multi-threaded here?

Reply:
To run the example which uses `#[tokio::main]`. We either need `rt-multi-thread` or I could replace it with `rt` and then use `#[tokio::main(flavor = "current_thread")]`